docs/source/executing-operations/demand-control.mdx

```diff
@@ -0,0 +1,554 @@
+
+To enable demand control in the router, configure the `preview_demand_control` option in `router.yaml`:
+
+```yaml title="Example demand control configuration"
+preview_demand_control:
+  enabled: true
+  mode: measure
+  strategy:
+    static_estimated:
+      list_size: 10 
+      max: 1000 
+```
+
+When `preview_demand_control` is enabled, the router measures the cost of each operation and can enforce operation cost limits, based on additional configuration.  
+
+Customize `preview_demand_control` with the following settings:
+
+| Option              | Valid values            | Default value | Description                                                                                          |
+| ------------------- | ----------------------- | ------------- | ---------------------------------------------------------------------------------------------------- |
+| enabled             | boolean                 | `false`       | Set `true` to measure operation costs or enforce operation cost limits.                              |
+| mode                | `measure`, `enforce`    |               | - `measure` collects information about the cost of operations.<br/>- `enforce` rejects operations exceeding configured cost limits |
+| strategy            | `static_estimated`      |               | `static_estimated` estimates the cost of an operation before it is sent to a subgraph                 |
+| static_estimated.list_size  | integer                 |               | The assumed maximum size of a list for fields that return lists.                                      |
+| static_estimated.max        | integer                 |               | The maximum cost of an accepted operation. An operation with a higher cost than this is rejected. |
+
+
+## Telemetry for demand control
+
+You can define router telemetry to gather cost information and gain insights into the cost of operations sent to your router:
+
+- Generate histograms of operation costs by operation name, where the estimated cost is greater than an arbitrary value.
+- Attach cost information to spans.
+- Generate log messages whenever the cost delta between estimated and actual is greater than an arbitrary value.
+
+### Instruments
+
+| Instrument      | Value | Default | Description                                                |
+| --------------- | ----- | ------- | ---------------------------------------------------------- |
+| cost.actual     |       |         | The actual cost of an operation, measured after execution.  |
+| cost.estimated  |       |         | The estimated cost of an operation before execution.        |
+| cost.delta      |       |         | The difference between the actual and estimated cost.       |
+
+### Attributes
+
+Attributes for `cost` can be applied to instruments, spans, and events—anywhere `supergraph` attributes are used.
+
+| Attribute      | Value | Default | Description                                                |
```

Comment:
Is it intentional default values are missing?

Reply:
removing empty cols